api/pyproject.toml

```diff
@@ -37,6 +37,7 @@ autopep8 = "^1.5.3"
 matplotlib = "^3.2.2"
 pytest-xdist = "^1.33.0"
 rope = "^0.17.0"
+docker-compose = "^1.26.2"
```

Comment:
what is the purpose of this python module?

Reply:
This way you don't have to install docker compose locally - it's installed by pip, so we have a consistent docker-compose across all our machines.
I ran into this when I got around to installing docker-compose on my ubuntu box, there isn't a nice packaged way to install it, and the only thing using docker-compose is this project - so it just makes sense.